README.md

```diff
@@ -16,296 +16,75 @@
+-   [Additional Information](./docs/additional-information.md)
+-   [Contribute to the Integration](./docs/contribute.md)
 
 ## Getting Started
 
 Follow the steps below to get started with the Databricks Integration quickly.
 
-**Install the integration**
-
-1. Follow the steps to [deploy the integration to a Databricks cluster](./docs/installation.md#deploy-the-integration-to-a-databricks-cluster).
-
-**Verify the installation**
-
-Allow the Databricks Integration a few minutes to begin sending data. Then,
-perform the following steps within New Relic:
-
-1. Open the [query builder](https://one.newrelic.com/data-exploration/query-builder).
-1. Click the button with the `+` symbol in the query builder tab bar to create a
-   new tab.
-1. In the query text field paste the following query:
-
-   `SELECT uniqueCount(executorId) AS Executors FROM SparkExecutorSample WHERE databricksclustername = '[YOUR_CLUSTER_NAME]'`
-
-   Replace the text `[YOUR_CLUSTER_NAME]` with the _name_ of the Databricks
-   [cluster](https://docs.databricks.com/en/getting-started/concepts.html#cluster)
-   where the integration was installed.
-
-   **NOTE:** If your cluster name has a `'` character, make sure to prefix the
-   `'` character with the `\` character. For example, if your cluster is named
-   "My Company's Cluster", replace `[YOUR_CLUSTER_NAME]` with
-   `My Company\'s Cluster`.
-
-1. Click on the button labeled "Run".
-
-When the Databricks Integration is installed and functioning properly, the
-result of the query will be a number greater than zero as in the screenshot
-below.
-
-![Successful Getting Started Query](./examples/getting-started.png)
+**1. Install the integration**
 
-**Next steps: Install the example dashboards**
+Follow the steps to [deploy the integration to a Databricks cluster](./docs/installation.md#deploy-the-integration-to-a-databricks-cluster).
 
-The following example dashboard JSON files are included in the [`./examples`](./examples/)
-directory that can be imported to create dashboards that showcase each of the
-main features of the Databricks Integration.
+**2. Verify the installation**
 
-* [Apache Spark](./examples/spark-daskboard.json)
-* [Databricks Job Runs](./examples/job-runs-dashboard.json)
-* [Databricks Pipeline Updates](./examples/pipeline-updates-dashboard.json)
-* [Databricks Queries](./examples/query-metrics-dashboard.json)
-* [Databricks Cluster Health](./examples/cluster-health-dashboard.json)
-* [Databricks Consumption & Cost](./examples/consumption-cost-dashboard.json)
-
-**NOTE:** Collection of data for some dashboards requires additional
-configuration. See the appropriate feature documentation for more details.
+Once the Databricks Integration has run for a few minutes, use the
+[query builder](https://one.newrelic.com/data-exploration/query-builder) in New
+Relic to run the following query (replace `[YOUR_CLUSTER_NAME]` with the _name_
+of the Databricks cluster _where the integration was installed_):
```

Comment:
@aso1124 we seem to have lost the comment about replacing `'` characters with `\'`.  The reason I included this is that when Databricks creates a personal compute cluster it defaults the name to "Foo Bar's Cluster" so it may be common for people to have a `'` in their cluster name.  Can we add this back?